### DISCUSSION TITLE
Keyboard focus

### DISCUSSION THREAD

**Discussion by yoshino-e-valley** [rdbende/Sun-Valley-ttk-theme#45] [Q&A] [answered]
Thank you for your work.
I have one request here.
When using the mouse, I can see where the focus is, but when I move with TAB, I don't know where the focus is (on Button).
Could you please improve it?

**Comment by rdbende** [comment, marked as answer]
Yes, it will be included in the next release, coming this weekend (I hope)

**Reply by yoshino-e-valley** [reply]
Thank you for your kindness.
